Makefile

```diff
@@ -267,6 +267,14 @@ minikube-deploy: minikube-start gadget-default-container kubectl-gadget
 btfgen:
 	+make -f Makefile.btfgen
 
+.PHONY: generate-manifests
+generate-manifests:
+	echo "---" > pkg/resources/manifests/deploy.yaml
+	echo "# This file is generated by 'make generate-manifests'; DO NOT EDIT." >> pkg/resources/manifests/deploy.yaml
+	cat pkg/resources/manifests/namespace.yaml >> pkg/resources/manifests/deploy.yaml
+	make -C charts APP_VERSION=latest template
```

Comment:
nit: should it use `IMAGE_TAG` instead of latest in case you're trying it on a different branch than main?

However it'll require to update the logic that checks if this is updated. (I did a quick try and break other things, so if this is difficult we can leave as it's)